apps/examples.nameguard.io/package.json

```diff
@@ -27,5 +27,6 @@
     "postcss": "^8",
     "tailwindcss": "^3.3.0",
     "typescript": "^5"
-  }
+  },
+  "version": null
```

Comment:
Appreciate your advice on why we make this `null`. Is it because the examples app isn't a true package? Even still maybe good to have a version on it if it would avoid the "null" in the `CHANGELOG.md`. Appreciate your advice.

Reply:
Removed and fixed changesets config.